utils/model_uploader/autotracing_utils.py

```diff
@@ -313,9 +314,14 @@ def prepare_files_for_uploading(
 
     # Delete model folder downloaded from HuggingFace during model tracing
     try:
-        shutil.rmtree(folder_to_delete)
+        if os.path.exists(folder_to_delete):
+            shutil.rmtree(folder_to_delete)
+            print(f"Successfully deleted temporary folder: {folder_to_delete}")
+        else:
+            print(f"Temporary folder not found, skipping deletion: {folder_to_delete}")
     except Exception as e:
-        assert False, f"Raised Exception while deleting {folder_to_delete}: {e}"
+        # Changed from AssertionError to a warning
+        print(f"Warning: Raised Exception while deleting {folder_to_delete}: {e}")
```

Comment:
Why do we want this?

Reply:
Got issue when in dev. Updated, no need this change anymore.